Formula/scm-manager.rb

```diff
@@ -66,4 +66,8 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    system "#{opt_libexec}/../bin/scm-cli-client version | grep #{version}"
```

Comment:
Thanks for adding a test! This should be a cleaner way to achieve a similar effect:

```ruby
assert_match version.to_s, shell_output("#{bin}/scm-cli-client version")
```